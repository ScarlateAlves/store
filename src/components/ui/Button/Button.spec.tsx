import { render } from "@testing-library/react";
import { Button } from ".";

it("renders the Button component", () => {
  const { container } = render(<Button>click</Button>);
  expect(container).toMatchSnapshot();
});
