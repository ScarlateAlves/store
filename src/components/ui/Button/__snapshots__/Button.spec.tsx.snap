// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`renders the Button component 1`] = `
<div>
  <button
    class="font-medium rounded cursor-pointer py-2 active:shadow-md active:shadow-gray-500/50 px-4 hover:opacity-90 active:opacity-70 transition-colors bg-[#DB4444] text-white text-base"
  >
    click
  </button>
</div>
`;
